package.json

```diff
@@ -36,8 +36,7 @@
     "lint:fix": "eslint --max-warnings 0 --config .eslintrc . --fix",
     "generate:assets": "npm run generate:readme:toc",
     "generate:readme:toc": "markdown-toc -i README.md",
-    "generate:docker": "docker build --tag asyncapi/server-api .",
-    "release": "npm run generate:docker && semantic-release"
+    "generate:docker": "docker build --tag asyncapi/server-api ."
```

Comment:
now I noticed that release workflows here are mixed with docker release. There are some additional changes that you need to do, sorry about that

```suggestion
    "docker:build": "docker build --tag asyncapi/server-api ."
```

you also need to add a new file, called `release-docker.yml` like below"
```yml
name: Release Docker Image
on: 
  release:
    types: 
      - published
jobs:

  publish-docker:
    name: Generating Docker
    runs-on: ubuntu-latest
    steps:
      - name: Checkout repository
        uses: actions/checkout@v3
      - name: Get version without v character
        id: version
        run: |
          VERSION=${{github.event.release.tag_name}}
          VERSION_WITHOUT_V=${VERSION:1}
          echo "value=${VERSION_WITHOUT_V}" >> $GITHUB_OUTPUT

      - name: Release to Docker
        run: | 
          echo ${{secrets.DOCKER_PASSWORD}} | docker login -u ${{secrets.DOCKER_USERNAME}} --password-stdin
          npm run docker:build
          docker tag asyncapi/server-api:latest asyncapi/server-api:${{ steps.version.outputs.value }}
          docker push asyncapi/server-api:${{ steps.version.outputs.value }}
          docker push asyncapi/server-api:latest

      - uses: meeDamian/sync-readme@82715041300710d9be7c726c9d6c683b70451087 #version 1.0.6 https://github.com/meeDamian/sync-readme/releases/tag/v1.0.6
        with:
          user: ${{secrets.DOCKER_USERNAME}}
          pass: ${{ secrets.DOCKER_PASSWORD }}
          slug: asyncapi/server-api
          description: CLI to work with your AsyncAPI files
```